assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -25,7 +25,7 @@ export const tests: Tests = {
         size: 1,
       },
     },
-    isActive: false,
+    isActive: true,
```

Comment:
Do we want to release the test?

Reply:
Yes when we merge

Reply:
Fair enough, maybe it is safer to merge it `off`, test in production that everything works well and then in another PR turn it on. Nevertheless, it is totally personal.

Reply:
I think you're right actually, I've changed it.